replicator-schema-translation/docs/index.rst

```diff
@@ -101,6 +101,13 @@ Run the Demo
    .. sourcecode:: bash
 
       docker-compose exec connect /etc/kafka/scripts/set_sr_modes_post_translation.sh
+      
+.. tip:: This demo showed a `one-time migration` of schemas across self-managed clusters. To configure a
+         `continuous migration`, the last steps would be to keep the origin (source) in READONLY mode, and set the destination
+         to READWRITE. Note that this only works for a "one-way" migration; that is, an active-to-passive
+         |crep| setup. This is especially helpful if configure topic names to be different on the destination,
```

Comment:
It's not clear how topic name configurations fit into the one-time vs continuous distinctions.  Can you please elaborate?